warewulf.spec.in

```diff
@@ -1,74 +1,107 @@
+%global wwgroup warewulf
+%{!?wwshared: %global wwshared %{_localstatedir}}
 %define debug_package %{nil}
 
 Name: warewulf
-Summary: A suite of tools for bare metal provisioning of containers
+Summary: A provisioning system for large clusters of bare metal and/or virtual systems
 Version: @VERSION@
-Release: 1%{?dist}
-License: BSD
-Source: https://github.com/hpcng/warewulf/archive/refs/tags/%{name}-%{version}.tar.gz
+Release: 1
```

Comment:
The `dist` tag should be used, if present, for Fedora/RHEL based distros.

Reply:
Done and thanks. I never thought about it since my build system replaces the whole release entry.